karma.conf.js

```diff
@@ -32,12 +29,14 @@ module.exports = function(config) {
         // preprocess matching files before serving them to the browser
         // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
         preprocessors: {
-            "tests/lib/linter.js": ["babel"]
+            "tests/lib/linter.js": ["webpack"]
         },
-        babelPreprocessor: {
-            options: {
-                presets: ["@babel/preset-env"]
-            }
+        webpack: {
+            mode: "none",
```

Comment:
seems it could be "development" | "production".

Reply:
I think there aren't too many differences between "development" and "none" in this case.
As for "production", this will cost more build time to minify the bundle. I'm not sure whether we need to do it or not.
Do you have any ideas?

Reply:
just curious. thanks for explaination! "development" | "none" sounds good to me.